go/arrow/internal/cdata_integration/entrypoints.go

```diff
@@ -0,0 +1,193 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build cdata_integration
+// +build cdata_integration
+
+package main
+
+import (
+	"C"
```

Comment:
don't need this since you already have the `import "C"` below

Reply:
Ah, right. Should I keep this one and remove the one below?

Reply:
keep this one. the separate `import` statement with the headers is the convention when using CGO so that it's explicit and doesn't get lost among the other imports.

In addition the `// #include` statements need to be *directly* above the `import` keyword, so it would make this block very awkward. So it's just cleaner keeping the one below.